common/reconciliation/store/blobstorewriter_test.go

```diff
@@ -0,0 +1,95 @@
+// The MIT License (MIT)
+//
+// Copyright (c) 2020 Uber Technologies, Inc.
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in all
+// copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+// SOFTWARE.
+
+package store
+
+import (
+	"context"
+	"testing"
+
+	"github.com/uber/cadence/common/blobstore"
+
+	"github.com/stretchr/testify/require"
+	"github.com/stretchr/testify/suite"
+
+	"github.com/uber/cadence/common/blobstore/filestore"
+	"github.com/uber/cadence/common/config"
+)
+
+type BlobstoreWriterSuite struct {
+	suite.Suite
+	*require.Assertions
+	blobstoreWriter *blobstoreWriter
+	uuid            string
+	extension       Extension
+	outputDir       string
+	blobstoreClient blobstore.Client
+}
+
+func TestBlobstoreWriterSuite(t *testing.T) {
+	suite.Run(t, new(BlobstoreWriterSuite))
```

Comment:
we are trying to avoid suites where possible and I believe we don't need it here.

Reply:
should I mock the client and write a standard unit test?